ports/espressif/boards/espressif_esp32s3_devkitc_1_n8r2/mpconfigboard.mk

```diff
@@ -5,6 +5,10 @@ USB_MANUFACTURER = "Espressif"
 
 IDF_TARGET = esp32s3
 
+CIRCUITPY_ESP_FLASH_SIZE = 8MB
 CIRCUITPY_ESP_FLASH_MODE = dio
 CIRCUITPY_ESP_FLASH_FREQ = 80m
-CIRCUITPY_ESP_FLASH_SIZE = 8MB
+
+CIRCUITPY_ESP_PSRAM_SIZE = 2MB
+CIRCUITPY_ESP_PSRAM_MODE = qio
+CIRCUITPY_ESP_PSRAM_FREQ = 80m
```

Comment:
Shouldn't this too be 40m? Please check w/ hardware.

Reply:
`chip_id` reports 40MHz on the `ESP32-S3-DevKitC-1-N8R2`

Reply:
The crystal is always 40Mhz, as far as I know.
Testing with a binary is the best way.
You load it up and do:
```py
import espidf
espidf.get_total_psram()
```
So long as this doesn't hang, and returns the size, the parameters are <= of the correct parameters.

Reply:
```
>>> import espidf
>>> espidf.get_total_psram()
2097152
```
QED?

Reply:
> Please check w/ hardware.

I did. Please assume I tested any Adafruit or Espressif settings changes. I have those boards.